### ISSUE TITLE
add link to determine your legislator and send message at any time

### ISSUE THREAD

**Issue by hwasserstrum** [chrisflesher/rcvfortexas#24]

**Comment by alexkarjeker** [comment]
I can take this up.

**Comment by ironunicorn** [comment]
So I found this civic information api that returns your representatives given your address: https://developers.google.com/civic-information/docs/v2/representatives/representativeInfoByAddress 

I was wondering if I could make this on our rcvfortexas.org website instead of having a link that redirects folks elsewhere. Let me know if it's cool to grab this issue. I don't want to step on anyone's toes. I'll be out until October 13th so I would be making the changes after that.

**Comment by hwasserstrum** [comment]
I tried to use the API that popped up but couldn't get it to work. If it actually returns the levels of offices like the link Alex found, I don't care which one we use. I don't know the status of what Alex was going to do. 
Here is what Alex sent to ***@***.***>UnsubscribeTo:chrisflesher/rcvfortexasCc:hwasserstrum,MentionWed, Aug 25 at 2:28 PM


@hwasserstrum - just to confirm the functionality, do you want this link on website? or something more

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
I responded that would work but haven't heard anything since then.ThanksHarriet
    On Tuesday, September 21, 2021, 08:52:57 PM CDT, Irene Foelschow ***@***.***> wrote:  
 
 


So I found this civic information api that returns your representatives given your address: https://developers.google.com/civic-information/docs/v2/representatives/representativeInfoByAddress

I was wondering if I could make this on our rcvfortexas.org website instead of having a link that redirects folks elsewhere. Let me know if it's cool to grab this issue. I don't want to step on anyone's toes. I'll be out until October 13th so I would be making the changes after that.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by ironunicorn** [comment]
Yes, it returns all the levels of representation the "Who Represents Me?" website provides. Do we need all of them? I was considering just showing the state level house and senate representatives.

**Comment by hwasserstrum** [comment]
My first thought was to include everyone someone votes for below President.  That way we could also use it when we try to identify local jurisdictions that might want to be early adopters. When I just used the link Alex sent, it included my State Board of Education representative but not my Houston Independent School District one or my Bellaire Mayor/City Council so it wouldn't serve that purpose. Maybe just include Congress and Texas State Rep/Senator since there already is legislation considered at those levels.ThanksHarriet
    On Wednesday, September 22, 2021, 08:04:46 AM CDT, Irene Foelschow ***@***.***> wrote:  
 
 


Yes, it returns all the levels of representation the "Who Represents Me?" website provides. Do we need all of them? I was considering just showing the state level house and senate representatives.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by alexkarjeker** [comment]
Hi all - I don't want to be a blocker here.

@Irene - happy to free up the task for you.

**Comment by ironunicorn** [comment]
Okay, thanks Alex! I'll get to it as soon as I'm back from my roadtrip.

**Comment by chrisflesher** [comment]
It seems like it would be really confusing to re-purpose unused pages...
hopefully there is a way to create a new custom file? NationBuilder support
usually responds in 24 hours.

**Comment by ironunicorn** [comment]
Oh! Yes, this is super helpful. Thanks @DJ-Eckam ! I created a custom template page `contact_elected_official` and I just now noticed it's in dropbox in the `sites` folder. So is it possible to just edit this file however I want and it'll load under https://www.rcvfortexas.org/contact_elected_offical?

I don't know why NationBuilder support didn't tell me about this...

**Comment by chrisflesher** [comment]
Congrats!

**Comment by ironunicorn** [comment]
@hwasserstrum I've got a preview of this page [here](https://rcvfortexas.nationbuilder.com/admin/sites/1/pages/79/preview?browser_id=default&logged_in=0&theme_id=5dabb2094445ea66232bca7e). Please let me know if you'd like me to tweak anything.

**Comment by hwasserstrum** [comment]
The page looks great but why is my State Senator the only one with an email address? The others just have phone number. Also, how does a website user find this page?ThanksHarriet
    On Wednesday, October 13, 2021, 02:48:14 PM CDT, Irene Foelschow ***@***.***> wrote:  
 
 


@hwasserstrum I've got a preview of this page here. Please let me know if you'd like me to tweak anything.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by hwasserstrum** [comment]
Here is where the link goes: https://wrm.capitol.texas.gov/home which is where your Who Represents Me link goes. If you click on the legislator's name you can easily see phone number and email address (or structured email message capability). With the link  you sent earlier today, I got to a dead end for several of my legislators without getting an email address.
    On Thursday, October 14, 2021, 10:41:52 AM CDT, Irene Foelschow ***@***.***> wrote:  
 
 


I can't follow your link. Is it Who Represents Me? ? When I go there it's just listing the official's name and title with a link to their official website.


| 
| 
|  | 
Who Represents Me?

Texas Legislative Council

Who Represents Me provides information about current districts and members of the Texas Senate, Texas House of R...
 |

 |

 |




On the webpage I made the displayed name also links to their official website. I could make it more button-y if it's not clear.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by ironunicorn** [comment]
Harriet and I chatted and right after the call I was able to reproduce her bug. I'm going to change the code so the links open in new tabs since this will fix the problem (and I think it'll also be a nicer user experience).
